apps/src/applab/designElements/ColorPickerPropertyRow.jsx

```diff
@@ -44,9 +44,13 @@ var ColorPickerPropertyRow = React.createClass({
       verticalAlign: 'top'
     };
     let colorPicker = this.state.displayColorPicker ? (
-      <ColorPicker type="sketch"
-                   color={this.state.value}
-                   onChangeComplete={this.handleColorChange}/>
+      <div style={{position: 'absolute'}}>
+        <div style={{position: 'fixed', top: '0', right: '0', bottom: '0', left: '0'}}
+             onClick={this.toggleColorPicker}></div>
```

Comment:
Instead of an invisible div over the screen, did you try the `handleClose` property on the color picker component [(see example)](http://casesandberg.github.io/react-color/#examples)?